app/src/components/governance/PageProposal.vue

```diff
@@ -22,7 +22,9 @@
               Proposed by {{ proposal.validator.name }}:
               <Address :address="proposal.proposer" />
             </template>
-            <template v-else-if="proposal.proposer !== `unknown`">
+            <template
+              v-else-if="proposal.proposer && proposal.proposer !== `unknown`"
```

Comment:
if this was a blank object or null this would be easier

Reply:
I can easily remove the `unknown` thing

Reply:
Removed